.travis.yml

```diff
@@ -25,4 +25,14 @@ deploy:
     name: Travis CI - Auto Doc Build
     on:
       branch: master
-      condition: $TRAVIS_EVENT_TYPE != cron
\ No newline at end of file
+      condition: $TRAVIS_EVENT_TYPE != cron
+
+notifications:
+  slack:
+    on_pull_requests: false
+    on_success: always
```

Comment:
we should probably drop `on_success` too, (to reduce the noice)

Reply:
@bdemers - I was just testing this PR out. Currently it's only posting to me privately on slack. Will remove it once it's working. (For some reason, I don't see the notifications on slack yet 😢)